documentation/docs/examples/timeline.mdx

```diff
@@ -0,0 +1,153 @@
+---
+id: timelinetool
+title: Timeline Tool
+describe: Applying styles to nodes based on date with Play & Stop
+---
+
+import { DemoWrapper } from '@site/docs/components/DemoWrapper';
+
+Reveal allows for applying styles to nodes based on date property and can be viewed as a playable entity
+where node styles are applied to different nodes which can be used to showcase progress of the model construction.
+
+
+:::note
+The test data used in the documentation site is for testing and to show the capabilities of the tool.
+The examples shown is therefore might not be a ideal model construction progression.
+:::
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+## Timeline
+
+The timeline tool provides ability to apply styles by date. It works by creating a set of "keyframes"  for provided dates.
+Each keyframe has a set of associated styled node collections.
+
+### Creating Keyframes
+To create Keyframe, use `createKeyframe(date: Date)`
+- `date` - Date reference for the Keyframe. Should provide the date value by `Date.now()`/`new Date('2021-10-05')`.
+
+### Add Node collection & Styles to Keyframes
+Tool provides feature to add nodes & styles to a Keyframe or number of Keyframes or to all of the Keyframes.
```

Comment:
Replace this with information about `Keyframe.(un)assignStyledNodeCollection` and provide a basic example for how to adding styles to the keyframe. This should also point to the documentation for the styling API for more information.